samples/Kconfig

```diff
@@ -14,6 +14,13 @@ config NCS_SAMPLES_DEFAULTS
 	help
 	  Use the default configuration for NCS samples.
 
+config NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE
```

Comment:
This new option should be documented in the empty app core sample and the Working with nRF53 user guide. If not now, then a task should be made to add this information.

Reply:
Let's hold off on this to avoid more noise in this PR. I've created NCSDK-5756.